libr/main/rvc2.c

```diff
@@ -104,6 +104,12 @@ R_API int r_main_rvc2(int argc, const char **argv) {
 			}
 		}
 		RCore *core = r_core_new ();
+		if (!core) {
+			free (message);
+			free (rp);
+			r_list_free (files);
+			return -11;
+		}
 		const char *author = r_config_get (core->config, "cfg.user");
```

Comment:
If u only need core to get the user just use the rsysuser call. Also core is never freed later